winui/AvatarView/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
```

Comment:
Description: Learn about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more.